src/librustc/util/profiling.rs

```diff
@@ -25,16 +25,28 @@ pub enum ProfilerEvent {
     GenericActivityEnd { category: ProfileCategory, time: Instant },
     QueryCacheHit { query_name: &'static str, category: ProfileCategory },
     QueryCount { query_name: &'static str, category: ProfileCategory, count: usize },
+    IncrementalLoadResultStart { time: Instant },
+    IncrementalLoadResultEnd { time: Instant },
```

Comment:
These should contain the query name too, I'd say.

Reply:
@michaelwoerister fixed